account_bank_statement_import_online_nordigen/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2022 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Online Bank Statements: Nordigen",
+    "version": "13.0.1.0.0",
```

Comment:
Hi @ChrisOForgeFlow, maybe you already noticed it, just making sure. The version should be 12.0! Thank you!

Reply:
Thanks, I solved